Fails to compile after update missing Monitor.hpp and MonitorState.hpp files.

I updated my system, came back to split monitor workspaces not working anymore, did a hyprpm update --force -v, which tells me that the files "hyprland/src/output/Monitor.hpp" and "hyprland/src/state/MonitorState.hpp" which are needed to build aren't existing.

Full error:
```
shell returned:  -> cd /run/user/1000/hyprpm/dasrainbow && PKG_CONFIG_PATH="/var/cache/hyprpm/dasrainbow/headersRoot/share/pkgconfig:$PKG_CONFIG_PATH" meson setup build --wipe
The Meson build system
Version: 1.11.1
Source dir: /run/user/1000/hyprpm/dasrainbow
Build dir: /run/user/1000/hyprpm/dasrainbow/build
Build type: native build
Project name: split-monitor-workspaces
Project version: 1.2.0
C++ compiler for the host machine: c++ (gcc 16.1.1 "c++ (GCC) 16.1.1 20260430")
C++ linker for the host machine: c++ ld.bfd 2.46.0
Host machine cpu family: x86_64
Host machine cpu: x86_64
Compiler for C++ supports arguments -std=c++23: YES
meson.build:8: WARNING: Consider using the built-in option for language standard version instead of using "-std=c++23".
Found pkg-config: YES (/usr/sbin/pkg-config) 2.5.1
Run-time dependency hyprland found: YES 0.55.4
Run-time dependency pixman-1 found: YES 0.46.4
Run-time dependency libdrm found: YES 2.4.134
Build targets in project: 1

Found ninja-1.13.2 at /usr/sbin/ninja

 -> cd /run/user/1000/hyprpm/dasrainbow && PKG_CONFIG_PATH="/var/cache/hyprpm/dasrainbow/headersRoot/share/pkgconfig:$PKG_CONFIG_PATH" meson compile -C build
ninja: Entering directory `/run/user/1000/hyprpm/dasrainbow/build'
[1/3] Compiling C++ object libsplit-monitor-workspaces.so.p/src_helpers.cpp.o
FAILED: [code=1] libsplit-monitor-workspaces.so.p/src_helpers.cpp.o
c++ -Ilibsplit-monitor-workspaces.so.p -I. -I.. -I/var/cache/hyprpm/dasrainbow/headersRoot/include -I/var/cache/hyprpm/dasrainbow/headersRoot/include/hyprland/protocols -I/var/cache/hyprpm/dasrainbow/headersRoot/include/hyprland -I/var/cache/hyprpm/dasrainbow/headersRoot/include/hyprland/src -I/usr/include/libdrm -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/pixman-1 -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O3 -std=c++23 -DWLR_USE_UNSTABLE -fno-gnu-unique -fPIC -MD -MQ libsplit-monitor-workspaces.so.p/src_helpers.cpp.o -MF libsplit-monitor-workspaces.so.p/src_helpers.cpp.o.d -o libsplit-monitor-workspaces.so.p/src_helpers.cpp.o -c ../src/helpers.cpp
In file included from ../src/helpers.txx:2,
                 from ../src/helpers.hpp:63,
                 from ../src/helpers.cpp:1:
../src/globals.hpp:7:10: fatal error: hyprland/src/output/Monitor.hpp: No such file or directory
    7 | #include <hyprland/src/output/Monitor.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[2/3] Compiling C++ object libsplit-monitor-workspaces.so.p/src_main.cpp.o
FAILED: [code=1] libsplit-monitor-workspaces.so.p/src_main.cpp.o
c++ -Ilibsplit-monitor-workspaces.so.p -I. -I.. -I/var/cache/hyprpm/dasrainbow/headersRoot/include -I/var/cache/hyprpm/dasrainbow/headersRoot/include/hyprland/protocols -I/var/cache/hyprpm/dasrainbow/headersRoot/include/hyprland -I/var/cache/hyprpm/dasrainbow/headersRoot/include/hyprland/src -I/usr/include/libdrm -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/pixman-1 -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O3 -std=c++23 -DWLR_USE_UNSTABLE -fno-gnu-unique -fPIC -MD -MQ libsplit-monitor-workspaces.so.p/src_main.cpp.o -MF libsplit-monitor-workspaces.so.p/src_main.cpp.o.d -o libsplit-monitor-workspaces.so.p/src_main.cpp.o -c ../src/main.cpp
../src/main.cpp:3:10: fatal error: hyprland/src/state/MonitorState.hpp: No such file or directory
    3 | #include <hyprland/src/state/MonitorState.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /usr/sbin/ninja -C /run/user/1000/hyprpm/dasrainbow/build
```

--- a/hyprpm.toml
+++ b/hyprpm.toml
@@ -43,7 +43,8 @@ commit_pins = [
     ["af923e30d1d24f1f4a4f5cb8308065173c1d9539", "ba22f56e4a5b42792d212816c722399fc47ce55e"], # 0.55.0
     ["a47147bc095e5b3be3eb8bd04f0ac242b968cd4d", "1f2ec598cb21c62fb403a5a6665a2478f83a4b7e"], # 0.55.1
     ["39d7e209c79d451efab1b21151d5938289da838d", "3aeb50c7fba3141590529b3f1a19dd80d1e77925"], # 0.55.2
-    ["fe5fe79a29ac3adaf3e75560b2f4b7a6d58b31c9", "6cff8cfffecf4729d1f5bacf82d96c5987f230f3"]  # 0.55.3
+    ["fe5fe79a29ac3adaf3e75560b2f4b7a6d58b31c9", "6cff8cfffecf4729d1f5bacf82d96c5987f230f3"], # 0.55.3
+    ["a0136d8c04687bb36eb8a28eb9d1ff92aea99704", "6cff8cfffecf4729d1f5bacf82d96c5987f230f3"]  # 0.55.4
 ]
 
 [split-monitor-workspaces]

